OpenOversight/app/templates/partials/links_subform.html

```diff
@@ -0,0 +1,7 @@
+<div>
+	<legend>{{form.links.label}}</legend>
+	{% for subform in form.links %}
+		{% include "partials/subform.html" %}
+	{% endfor %}
+	<button class="btn btn-success js-add-another-button" disabled>Add another link</button>
```

Comment:
Similarly here if I click "Remove" if there is only one link, then "Add another link" doesn't do anything